parts/kubernetesmastercustomdata.yml

```diff
@@ -19,25 +19,19 @@ write_files:
   content: |
     [Service]
     ExecStart=
-    ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay --log-driver=journald
+    ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay
 
-- path: "/etc/systemd/journald.conf.d/kubernetes.conf"
-  permissions: "0644"
-  owner: "root"
-  content: |
-    [Journal]
-    Storage=persistent
-    ForwardToSyslog=no
-    MaxLevelConsole=warning
-    RateLimitInterval=1s
-    RateLimitBurst=20000
-  
 - path: "/etc/docker/daemon.json"
   permissions: "0644"
   owner: "root"
   content: |
     {
-      "live-restore": true
+      "live-restore": true,
```

Comment:
@anhowe The settings below will constrain Docker logs to 50m, with a max of 5 files. 👍